com.gigitux.youp.json

```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "com.gigitux.youp",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.38",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "youp",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--talk-name=org.freedesktop.Notifications",
+    "--device=dri",
+    "--socket=pulseaudio",
+    "--filesystem=home"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "env": {
+      "CARGO_HOME": "/run/build/youp/cargo"
+    }
+  },
+  "modules": [
+    "shared-modules/libappindicator/libappindicator-gtk3-introspection-12.10.json",
+    {
+      "name": "youp",
+      "buildsystem": "meson",
+      "config-opts": ["-Dbuildtype=release"],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/gigitux/youpforwhatsapp/releases/download/2.0/youp-2.0.0.tar.xz",
+          "sha256": "1019a0d497b2c36f94a6e772a6eb3da6ebf818cd35944f0b0df609b13ac9850e"
+        }
+      ],
+      "post-install": [
+        "install -Dm 644 ./data/resources/128x128/com.gigitux.youp.png /app/share/app-info/icons/flatpak/64x64/com.gigitux.youp.png"
```

Comment:
I think that's the problem here. You shouldn't copy icon in app-info.

Also it will fail because the current directory is a build directory so the path would be `../data/resources/128x128/com.gigitux.youp.png`

Reply:
Also copied into 64x64, so the 128x128 icon isn't found.

Reply:
Removing this post-install step and the icons are in the right place.

Reply:
Mmm, I have added this later. It was an attempt, even removing this line CI fails